src/tools/ScenarioMeasurement/Startup/DeviceTimeToMain.cs

```diff
@@ -0,0 +1,66 @@
+﻿using Microsoft.Diagnostics.Tracing;
+using Reporting;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+using System.Text.RegularExpressions;
+
+namespace ScenarioMeasurement
+{
+
+    /// <summary>
+    /// This is a custom parser that does not enable any profiling. Instead, it relies on being passed nettrace files collected
```

Comment:
I am wondering if instead of using this we should pass a path to a folder, and a Regex of all of the files to match of that type. I think I would prefer that to having magic that looks for certain values.

Reply:
Implemented a parse method specifically for parsing multiple files from a directory.